elements/pfe-tabs/src/pfe-tabs.js

```diff
@@ -259,8 +259,9 @@ class PfeTabs extends PFElement {
     if (tabIndexFromURL > -1) {
       this._setFocus = true;
       this.selectedIndex = tabIndexFromURL;
-    } else if (this.selectedIndex === null) {
-      this.selectedIndex = 0;
+    } else {
```

Comment:
If we have arrived here then this.selectedIndex might be set to null. If that's the case, we need to explicitly get the tab index. noice.